src/concurrency/thread/threadpool-walk.md

```diff
@@ -9,22 +9,15 @@ the current directory and calls [`execute`] to perform the operations of reading
 and computing SHA1 hash.
 
 ```rust,no_run
-# #[macro_use]
-# extern crate error_chain;
+#![feature(rustc_private)]
```

Comment:
I don't think we need rustc_private for returning a result in main.  Is there another reason for this?

Reply:
I was writing that code on my local install and there it was needed. It's absolutely possible that it's not needed here. I'll check it right away.